webapp/blog/views.py

```diff
@@ -44,8 +45,15 @@ def homepage():
         filter = None
 
     try:
+        get_page = 1
```

Comment:
A little comment here would be nice to explain that we offset because of newsletter card

Reply:
Ah, thought I'd added one :+1: